### ISSUE TITLE
Consider to move exporter into blackbox-exporter repo?

### ISSUE THREAD

**Issue by v-zhuravlev** [adinhodovic/blackbox-exporter-mixin#7]
Hi, many exporters have its mixin in the same repo as the exporter itself:

https://github.com/prometheus/node_exporter/tree/master/docs/node-mixin
https://github.com/prometheus/mysqld_exporter/tree/main/mysqld-mixin
and so on...

WDYT about proposing this mixin into blackbox-exporter repo for discoverability and involving other contributors? TY.

**Comment by adinhodovic** [comment]
Yeah, I'm aware - maintaining one here: https://github.com/danihodovic/celery-exporter/tree/master/celery-mixin, https://github.com/danihodovic/borg-exporter/tree/master/borg-mixin.

As of now not done it as I've got multiple of them that are work in progress and needs some cleanups:

- https://github.com/adinhodovic/ingress-nginx-mixin
- https://github.com/adinhodovic/rabbitmq-mixin
- https://github.com/adinhodovic/blackbox-exporter-mixin

Will do it soon-ish when I find time.
